lib/raidenclient/RaidenClient.ts

```diff
@@ -4,6 +4,25 @@ import assert from 'assert';
 import BaseClient, { ClientStatus } from'../BaseClient';
 import errors from './errors';
 
+/**
+ * A utility function to parse the payload from an http response.
+ */
+async function parseResponseBody(res: http.IncomingMessage): Promise<any> {
```

Comment:
it's possible to add a generic type for the body parsing and apply it to the return type (instead of `any`)